.scripts/automation_generate.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+# rush install
+npm install -g @microsoft/rush
+rush install
+# generate
+npm install -g azure-track2-js-sdk-release-tools
```

Comment:
Move the scripts to init_script so that it would be called only once.

Reply:
I'll test the initOptions

Reply:
fixed